torch/csrc/jit/pybind_utils.h

```diff
@@ -91,10 +91,8 @@ inline c10::optional<TypePtr> unifyOrInitializeType(
 inline TypedIValue toTypedIValue(py::handle input) {
   if (THPVariable_Check(input.ptr())) {
     auto ten = py::cast<at::Tensor>(input);
-    if (ten.is_sparse()) {
-      AT_ERROR("sparse tensors not supported");
-    }
-    return TypedIValue(ten, CompleteTensorType::create(ten));
+    auto complete = CompleteTensorType::create(ten.scalar_type(), ten.device(), ten.sizes());
```

Comment:
Also leaking the details of how to create a CompleteTensorType. This will be nearly impossible to find if we change complete tensor type.